### DISCUSSION TITLE
Laggy pair programming

### DISCUSSION THREAD

**Discussion by bkilshaw** [zed-industries/zed#17775] [Feature Requests]
A co-worker and I spend a lot of time pair programming and Zed worked great for a long time. However, the last couple of months it's been super laggy. Keystrokes can sometimes take minutes to show up and that seems to also break the diff so things just randomly break. It's feels like adversarial pair programming at times. 

Has anyone else experienced this? Every update I hope it's going to get fixed but nothing yet.

**Comment by hugodzibetti** [comment]
Yes

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
